bootcamp/settings.py

```diff
@@ -456,6 +457,14 @@ def get_var(name, default):
 CYBERSOURCE_REFERENCE_PREFIX = get_var("CYBERSOURCE_REFERENCE_PREFIX", None)
 
 
+# FluidReview
+FLUIDREVIEW_ACCESS_TOKEN = get_var("FLUIDREVIEW_ACCESS_TOKEN", None)
```

Comment:
If these should be required can you add a validation check for these variables?

Reply:
There aren't validation checks for any required settings/variables yet - the app starts up without errors when I clear out my `.env` file. Maybe this should be a separate issue/PR?

Reply:
That's fine, I filed https://github.com/mitodl/bootcamp-ecommerce/issues/146